dymension/chain.json

```diff
@@ -495,6 +521,12 @@
     ]
   },
   "explorers": [
+    {
```

Comment:
Please insert these additions to the bottom of their respective arrays

Reply:
done